torch/nn/modules/sparse.py

```diff
@@ -206,6 +206,9 @@ class EmbeddingBag(Module):
         sparse (bool, optional): if ``True``, gradient w.r.t. :attr:`weight` matrix will be a sparse tensor. See
                                  Notes for more details regarding sparse gradients. Note: this option is not
                                  supported when ``mode="max"``.
+        include_last_offset (bool, optional): if ``True``, :attr:`offsets` has one additional element, where the last element
+                                      is equivalent to the size of `indices`. This matches the CSR format. Note:
+                                      this option is currently only supported when ``mode="sum"``.
```

Comment:
we should support it for other options as well (even if it's not efficient yet). Can we add implementation to slice off the last element for other modes for now?